core/commands/files/files.go

```diff
@@ -249,10 +249,10 @@ Examples:
 `,
 	},
 	Arguments: []cmds.Argument{
-		cmds.StringArg("path", false, false, "Path to show listing for. Defaults to '/'."),
+		cmds.StringArg("path", false, false, "Path to show listing for.").Default('/'),
```

Comment:
strings in go use double quotes. The single quote here will be interpreted as a character and fail at runtime if someone uses the default values.